.github/workflows/zeebe-testbench.yaml

```diff
@@ -62,7 +75,7 @@ jobs:
       - uses: ./.github/actions/build-platform-docker
         id: build-zeebe-docker
         with:
-          repository: gcr.io/zeebe-io/zeebe
+          repository: europe-docker.pkg.dev/camunda-saas-registry/zeebe/zeebe
```

Comment:
```suggestion
          repository: europe-docker.pkg.dev/camunda-saas-registry/zeebe-io/zeebe
```

Reply:
Are you sure? I can access europe-docker.pkg.dev/camunda-saas-registry/zeebe/zeebe but not europe-docker.pkg.dev/camunda-saas-registry/zeebe-io/zeebe

Reply:
Yes, you can try again.
It will work now.